specs/nativescript/add_platform.yaml

```diff
@@ -0,0 +1,14 @@
+name: Configure your current NativeScript project to target the selected platform.
+command: "ns platform add {{platform}}"
+tags:
+  - ns
+  - tns
```

Comment:
Hey @erodriguezh! Thanks so much for your contribution. One thing i'm noticing is that we have duplicated tags for `ns`, `tns`, and `nativescript` here. Would it be possible to consolidate and just have the `nativescript` tag instead?